AirCasting/StreamSynchronizer/StreamRemover.swift

```diff
@@ -0,0 +1,90 @@
+// Created by Lunar on 22/11/2021.
+//
+
+import Foundation
+
+protocol StreamRemover {
+    func deleteStreams(client: APIClient, session: SessionEntity, completion: @escaping () -> Void)
```

Comment:
`APIClient` should not be passed as a param, it should be a dependency of a concrete implementation (`StreamRemoverDefault`)

Reply:
It appeared it is not needed 😄